python/cudf_polars/tests/test_config.py

```diff
@@ -654,6 +656,18 @@ def test_ir_execution_context_from_config_options(
     context.get_cuda_stream()  # no exception
 
 
+def test_cuda_stream_pool():
+    pool_config = CUDAStreamPoolConfig()
+    pool = pool_config.build()
+
+    assert pool.get_pool_size() == 16
```

Comment:
Is this testing a default from a different library (i.e. rapidsmpf). Maybe not worth the fragility?